hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1854,7 +1855,7 @@ public OzoneDataStreamOutput createMultipartStreamKey(
             .setMultipartUploadID(uploadID)
             .setIsMultipartKey(true)
             .enableUnsafeByteBufferConversion(unsafeByteBufferConversion)
-            .setConfig(conf.getObject(OzoneClientConfig.class))
+            .setConfig(clientConfig)
```

Comment:
This one, and the other two similar items are creating new configs intentionally.  Please see HDDS-8216.

To reduce config loading, but still keep config loading to minimum, we could cache and reuse `OzoneClientConfig` instance per EC chunk size.

Reply:
I understand, but here we are using `Configuration` to pass parameters, which shouldn't be the role of `Configuration`. If a parameter needs to be computed at runtime, it should not be directly obtained from Configuration. Therefore, I added a `StreamBufferArgs` to modify and pass the StreamBuffe related parameters . 

IMO, Configuration should ideally not provide a direct `set` method. Modifications to Configuration should only occur in "test code" and during "reconfiguration". Otherwise, Configuration should be read-only.